http/1.9/modules/ROOT/pages/http-connector-config-acb.adoc

```diff
@@ -0,0 +1,252 @@
+. Select the *Advanced* tab to configure the following attributes:
+* *Allowed methods*
++
+Comma-separated list of allowed HTTP methods by this listener. To allow all methods, do not define this attribute.
++
+* *Response streaming mode*
++
+Defines whether or not to use streaming when the response is sent.
++
+* *Output MIME type*
++
+Defines the format of the response.
++
+* *Parameters* 
++
+Add additional parameters for the *Listener*.
++
+* *Output encoding*
++
+Select the encoding to transform the data to.
++
+* *Streaming Strategy*
++
+This attribute is not currently enabled in the configuration UI. Configure it in the XML editor. See xref:connectors::streaming-strategies.adoc[]
++
+* *Primary node only*
++
+Check this box to execute the *Listener* trigger on the primary node only when running in a cluster.
+* *Redelivery policy*
++
+This attribute is not currently enabled in the configuration UI. Configure it in the XML editor. See xref:http-documentation.adoc#RedeliveryPolicy[Redelivery Policy].
+* *Connection*
+** *Reconnection strategy*
++
+This attribute is not currently enabled in the configuration UI. Configure it in the XML editor. See xref:http-documentation.adoc#reconnect[Reconnect] and xref:http-documentation.adoc#reconnect[Reconnect Forever].
+* *Reponse*
++
+See xref:http-documentation.adoc#HttpResponseAttributes[HTTP Respoonse Attributes]
++
+* *Error Response*
++
+An HTTP error response returns a status code for the error. 
++
+** *Body*
++
+The body of the error message.
+** *Headers*
++
+Headers to include in the error message.
+** *Status code*
++
+The status code of the error response, for example `500`.
+** *Reason phrase*
++
+Reason for the error, for example `Gateway Timeout` or `Internal Server Error`.
+
+For details about the Listener configuration attributes, see xref:http-documentation.adoc#listenerConfig[Listener Configuration].
+
+== Configuring the Attributes for Basic Security Filter
+
+The *Basic security filter* component authenticates received HTTP requests. Use this component after a *Listener* component in your integration flow.
+
+. Click the *Basic security filter* component on the canvas.
++
+The configuration UI opens.
+. In the *General* tab, configure the following attributes:
+* *Realm*
++
+Authentication realm.
++
+*Security providers*
++
+This attribute is not currently enabled in the configuration UI. Configure it in the XML editor. See xref:http-documentation.adoc#basicSecurityFilter[Basic Security Filter].
++
+* *Attributes*
++
+HTTP Request attributes that come from an HTTP Listener source and check the `Authorization` header. 
+
+
+=== Configuring Attributes for the HTTP Request
+
+The Anypoint Connector for HTTP (HTTP Connector) *Request* operation enables you to consume an external HTTP service using the default `GET` method. Otherwise, you can choose what methods the operation accepts
```

Comment:
```suggestion
The Anypoint Connector for HTTP (HTTP Connector) *Request* operation enables you to consume an external HTTP service using the default `GET` method. Otherwise, you can choose which methods the operation accepts.
```